packages/@react-spectrum/color/docs/ColorArea.mdx

```diff
@@ -0,0 +1,310 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import colorTypes from 'docs:@react-types/color/src/index.d.ts';
+import docs from 'docs:@react-spectrum/color';
+import {HeaderInfo, PropTable, TypeLink, PageDescription} from '@react-spectrum/docs';
+import {Layout} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/color/package.json';
+import statelyDocs from 'docs:@react-stately/color';
+
+export default Layout;
+
+```jsx import
+import {ColorArea, ColorSlider, ColorWheel} from '@react-spectrum/color';
+import {Flex, Grid} from '@react-spectrum/layout';
+import {Label} from '@react-spectrum/label';
+import {View} from '@react-spectrum/view';
+```
+
+---
+category: Color
+keywords: [color area]
+---
+
+# ColorArea
+
+<PageDescription>{docs.exports.ColorArea.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorArea']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/color-area/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ColorArea defaultValue="#7f0000" />
+```
+
+## Value
+
+A ColorArea requires either an uncontrolled default value or a controlled value, provided using the `defaultValue` or `value` props respectively.
+The value provided to either of these props should be a color string or <TypeLink links={colorTypes.links} type={colorTypes.exports.Color} /> object.
+`xChannel` and `yChannel` props may also be provided to specify which color channels the color area should display, and the direction of each channel in the color gradient.
+This must be one of the channels included in the color value, for example, for RGB colors, the "red", "green", and "blue" channels are available.
+For a full list of supported channels, see the [Props](#props) table below.
+
+In the example below, the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.parseColor} /> function is used to parse the initial color from a HSL string
+so that `value`'s type remains consistent.
+
+```tsx example
+import {parseColor} from '@react-stately/color';
+
```

Comment:
Would be nice to include the Flex and Label imports here for users to copy paste